adapter-rest-contract/src/main/resources/io/github/fileanalysissuite/adapters/rest/contract/swagger.yaml

```diff
@@ -145,21 +184,33 @@ components:
           type: string
         title:
           type: string
+          nullable: true
```

Comment:
Are we trying to distinguish between being `null` and not being present?

Reply:
No, this is to improve the code generation. This property allows nulls and the generated code now has `string?`